### PR TITLE
LPS-159900 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-159900

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro [hunk 1]
```diff
@@ -864,6 +864,41 @@ definition {
 		AssertVisible(locator1 = "Icon#DELETE");
 	}
 
+	macro selectAssetsToDisplayOnWidget {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro:867]: Can you select a folder? If article only please rename to selectArticlesToDisplayOnWidget for clarity

Comment (austinchiang) [reply portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro:867]: Oh nevermind I see https://github.com/liferay-lima/liferay-portal/pull/3795/commits/eff355a316b5f9fc0f8fa3ffbf479f71c02d39cf#diff-69c24ae27b612c28e85d0563fecaecb9cbd92e269ca87140fb38a2bf02604c7fR887-R892 now

File: portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro [hunk 2]
```diff
@@ -871,6 +871,20 @@ definition {
 
 		SelectFrame(locator1 = "IFrame#MODAL_BODY");
 
+		if (isSet(viewSectionsInModal)) {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro:874]: change to viewOpenSectionsInModal

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please make changes and resend

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Review by austinchiang — APPROVED** [review]

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**